docs/citations.rst

```diff
@@ -25,24 +25,24 @@ Use Citations
 ~~~~~~~~~~~~~
 
 .. bibliography:: bib/use_citations.bib
-   :list: bullet
+   :list: enumerated
    :all:
-   :style: plain
+   :style: unsrt
```

Comment:
I've kept the `:style:` option here even though I've added `bibtex_default_style = "unsrt"` to `conf.py` in case we change things in the future these will stay as `unsrt`.